packages/hooks/useNSNameInfo.ts

```diff
@@ -103,11 +117,13 @@ export const useNSNameInfo = (
   networkId: string | undefined,
   tokenId: string | null | undefined,
   enabled?: boolean,
+  // We can provide address for case where gno user has no tokenId/username but has profile info
+  gnoAddress?: string,
 ) => {
   const { data: nsInfo, ...other } = useQuery(
     nsNameInfoQueryKey(networkId, tokenId),
     async () => {
-      if (!tokenId) return null;
+      if (!tokenId && !gnoAddress) return null;
```

Comment:
I don't link this change, we should avoid network-specific things in interfaces

gno uses the concept of `AddressOrName`, maybe we should accept this instead of `tokenId` here